kb/all-the-ways-to-run-a-cluster.md

```diff
@@ -0,0 +1,123 @@
+However, the right way for you depends entirely on your use case and where you plan to run it.
+This article aims to maintain a comprehensive list of all the ways we know of.
+
+<!-- truncate -->
+
+## Temporalite
+
+[Temporalite](https://github.com/temporalio/temporalite/cmd/temporalite) is a complete, but minimal, Temporal Cluster distribution (four Temporal Server services and a database) that runs as a single process with zero runtime dependencies.
+It supports persistence to disk and in-memory mode through SQLite.
+
+Temporalite is great for local testing and development purposes.
+It offers great performace in terms of processing Workflow Executions per second.
+Although it does currently default to one History Shard, we plan to make this setting configurable at start time, and you can adjust the setting by cloning the repository.
+In theory, performance at this point is limited to your machine's processing capabilities, SQLite capacity, and SQLite read/write speeds.
+
+One drawback is that Temporalite does not yet support [Advanced Visibility](/visibility/#advanced-visibility), or other Cluster features such as Archival.
+
+Temporalite also requires that you have Go 1.18 or later installed.
+
+To build and start Temporalite, run the following commands:
+
+```bash
+go install github.com/temporalio/temporalite/cmd/temporalite@latest
+temporalite start --namespace default
+```
+
+For macOS users, if you receive the error `error setting up schema: stat /Users/<user_name>/Library/Application Support/temporalite/db: no such file or directory`, create the `temporalite/db` directories in your `Application Support` library and run the `start` command again.
+
+Replace `default` with your custom [Namespace](/namespaces) name.
+
+Local Temporal Clients and Workers can connect to Temporalite at 127.0.0.1:7233 (default Client connection for most SDKs) and the Temporal Web UI at 127.0.0.1:8233.
+
+## Docker & Docker Compose
+
+You can easily run a Temporal Cluster in [Docker](https://docs.docker.com/engine/install) containers using [Docker Compose](https://docs.docker.com/compose/install).
+
+If you have Docker and Docker Compose installed, all you need to do is clone the [temporalio/docker-compose](https://github.com/temporalio/docker-compose) repo and run the `docker-compose up` command from its root.
+
+The `temporalio/docker-compose` repo comes loaded with a variety of configuration templates that enable you to try all three databases that the Temporal Platform supports (PostgreSQL, MySQL, Cassandra).
+It also enables you to try [Advanced Visibility](/visibility/#advanced-visibility) using [Search Attributes](/visibility/#search-attribute), emit metrics, and even play with the [Archival](/clusters/#archival) feature.
+
+Running your Cluster in Docker is convenient and enables you to play with features.
+However, it does not offer the same performace in terms of processing Workflow Executions per second as Temporalite.
```

Comment:
Is this only true if you run this on your local machine?

Reply:
Added a clarification that it is "usually the case for local machines"